core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -94,5 +98,18 @@ var ExplorationEditorTranslationTab = function() {
     _selectLanguage(language);
     waitFor.pageToFullyLoad();
   };
+
+  this.navigateToTranslationTabFeedback = function() {
+    waitFor.elementToBeClickable(
+      TranslationTabFeedbackButton,
+      'Feedback tab of translation page is not clickable');
+    TranslationTabFeedbackButton.click();
+    waitFor.pageToFullyLoad();
+  };
+
+  this.expectActiveTabToRemainSame = function() {
+    expect(element(by.css('.protractor-test-translation-feedback-tab'))[0]
```

Comment:
Do we have to add an index here (`[0]`)? (ditto below)

Reply:
Yes, we do.
It won't work otherwise.
A function cannot be compared with another function.